package.json

```diff
@@ -1,40 +1,45 @@
 {
   "name": "@hetznercloud/ngx-translate-mock",
-  "version": "0.2.0",
+  "version": "1.0.0",
   "license": "MIT",
   "repository": "https://github.com/hetznercloud/ngx-translate-mock",
   "scripts": {
     "ng": "ng",
-    "build": "ng build && yarn copy:readme",
-    "lint": "ng lint",
-    "pretty": "prettier --list-different --config ./.prettierrc \"**/*.@(js|ts|scss|md|json|yml|html)\"",
-    "pretty:write": "prettier --write --config ./.prettierrc \"**/*.@(js|ts|scss|md|json|yml|html|template)\"",
```

Comment:
Seems to be missing?

Reply:
Good catch -- added prettier back in and reformatted the code.